app/code/Magento/Bundle/view/frontend/templates/catalog/product/view/summary.phtml

```diff
@@ -6,6 +6,7 @@
 ?>
 
 <?php
+// phpcs:disable PHPCompatibility.Miscellaneous.RemovedAlternativePHPTags.MaybeASPOpenTagFound
```

Comment:
Could you remove useless opening and closing PHP tags just after the copyright?

Reply:
fixed